src/booking/booking.controller.ts

```diff
@@ -0,0 +1,37 @@
+import { Controller, Get, Param, UseGuards } from '@nestjs/common';
+import {
+  ApiBearerAuth,
+  ApiExtraModels,
+  ApiOkResponse,
+  ApiOperation,
+  ApiTags,
+  refs,
+} from '@nestjs/swagger';
+import { UserGuard } from 'src/common/guards/user.guard';
+import { BookingService } from './booking.service';
+import { DatesAvailableResDto, DatesNotAvailableResDto } from './dto';
+
+@Controller('booking')
+@UseGuards(UserGuard)
+@ApiTags('BOOKING')
+@ApiBearerAuth()
+export class BookingController {
+  constructor(private readonly bookingService: BookingService) {}
+
+  @Get('/availability/:id')
+  @ApiOperation({ summary: 'GET ACCOMODATION AVAILABILITY' })
```

Comment:
Please, use lowercase to identify tags for swagger.
They were fixed in another branch. 
```js
@ApiTags('booking')
```
```js
@ApiOperation({ summary: 'Get accommodation availability' })
```

Reply:
fixed